### ISSUE TITLE
Unexpected nooelint behavior with oelint.file.underscores

### ISSUE THREAD

**Issue by gmcn42** [priv-kweihmann/oelint-adv#904]
In a .bb inheriting kernel-fit-image.bbclass (see https://github.com/priv-kweihmann/oelint-adv/pull/903), we tried deactivating the underscores check after seeing

```console
meta-mylayer/recipes-images/fitimage/my-fitimage.bb:1:error:oelint.file.underscores:Filename should contain at least one '_' in the end
```

with a `# nooelint: oelint.file.underscores` in the first line of the file. This has worked for us with other suppressions, like mandatoryvar.SRC_URI. However, with underscores - while the original finding is gone - we're then seeing

```console
meta-mylayer/recipes-images/fitimage/my-fitimage.bb:1:info:oelint.file.inlinesuppress_na:Inline suppression for 'oelint.file.underscores' is not needed
```

**Comment by priv-kweihmann** [comment]
Yeah that looks like a bug to me.
It should report the finding on line 0, but actually uses line 1.

Will fix that in the next couple of days
